src/mixins/legal-api-mixin.ts

```diff
@@ -29,11 +29,10 @@ export default class LegalApiMixin extends Vue {
       .then(response => {
         if (response && response.data) {
           return response.data.filing
-        } else {
```

Comment:
I removed the else case here (and below) as it's redundant and the new code is cleaner-looking (and not any slower).